SignalMessaging/utils/DebugLogger.m

```diff
@@ -71,9 +75,10 @@ - (void)wipeLogs {
     }
 }
 
-- (NSString *)logsDirectory {
-    NSArray *paths          = NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES);
-    NSString *baseDir       = ([paths count] > 0) ? [paths objectAtIndex:0] : nil;
+- (NSString *)logsDirectory
+{
+    NSArray *paths = NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES);
```

Comment:
I've opened a card about where logging should take place.  Presumably our debug log uploads should include SAE logs.  Should SAE only log to shared data?  Should main app too?  Do we need to create a new protected directory for this?